cmd/antrea-controller/controller.go

```diff
@@ -87,6 +87,8 @@ func run(o *Options) error {
 		appliedToGroupStore,
 		networkPolicyStore)
 
+	endpointQueryReplier := networkpolicy.NewEndpointQueryReplier(networkPolicyController)
```

Comment:
It seems it doesn't really need a concrete instance here, all consumers just need the interface, right? Maybe you could use interface here, so the struct name can be simplified as the lower case of the interface, i.e. endpointQuerier?

Reply:
Fixed in https://github.com/vmware-tanzu/antrea/pull/1116